### PR TITLE
Correlation proof

### PR DESCRIPTION
**Pull request by paulbakker65**
Proofs the stability increase

### CODE REVIEW

File: src/main/matlab/correlation.m [hunk 1]
```diff
@@ -0,0 +1,72 @@
+ges = std(Gene_Expression);
```
Comment (Delfil) [inline src/main/matlab/correlation.m]: missing docs

File: src/main/matlab/correlation.m [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+p_array(t) = p;
+total_mean_1(t) = mean(std_1);
+total_mean_avg(t) = mean(std_avg);
+
+end
+%subplot(2,1,1);
+%histogram(std_1)
+%subplot(2,1,2);
+%histogram(std_avg)
+
```
Comment (Delfil) [inline src/main/matlab/correlation.m]: commented code

File: src/main/matlab/correlation.m [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+total_acc_1 = zeros(1,numRuns*numGenes);
+total_acc_avg = zeros(1,numRuns*numGenes);
+counter_acc = 1;
+
+for t = 1:numRuns
+counter = 1;
+std_1 = zeros(1,numGenes);
+std_avg = zeros(1,numGenes);
+genes = randperm(size(absCorr,1),numGenes);
+
```
Comment (Delfil) [inline src/main/matlab/correlation.m]: format missing

File: src/main/matlab/plotting_correlation.m [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+h(1) = subplot(2,1,1);
```
Comment (Delfil) [inline src/main/matlab/plotting_correlation.m]: doc missing

File: src/main/matlab/threshold.m [hunk 5]
```diff
@@ -0,0 +1,16 @@
+function t = threshold(act, label)
```
Comment (Delfil) [inline src/main/matlab/threshold.m]: doc missing

File: src/main/matlab/plotting_correlation.m [hunk 6]
```diff
@@ -0,0 +1,21 @@
+%Here we plot the p-values of both t-test we ran 100 times.
+figure;
+boxplot(-log(p_array));
```
Comment (Delfil) [inline src/main/matlab/plotting_correlation.m]: Description what the whole file does is missing

**Comment by Delfil** [comment]
Ok

**Comment by bottervanger** [comment]
OK

**Pull request merged**